languages/es.json

```diff
@@ -0,0 +1,245 @@
+{
+	"bh2e": {
+		"buttons": {
+			"roll": "Tirada"
+		},
+		"dialogs": {
+			"titles": {
+				"attackRoll": "Tirada de ataque"
+			}
+		},
+		"dice": {
+			"one": "1",
+			"d4": "1d4",
+			"d6": "1d6",
+			"d8": "1d8",
+			"d10": "1d10",
+			"d12": "1d12",
+			"d20": "1d20",
+			"special": "Otro (Consulta las Capacidades)"
+		},
+		"fields": {
+			"headers": {
+				"abilities": "Capacidades",
+				"armour": "Armadura",
+				"background": "Descripción",
+				"equipment": "Equipo",
+				"weapons": "Armas"
+			},
+			"labels": {
+				"abilities": "Capacidades",
+				"armedDamage": "Daño con armas",
+				"armourValue": "Valor de protección",
+				"attacks": "Ataques",
+				"attribute": "Características",
+				"attributes": {
+					"charisma": {
+						"long": "Carisma",
+						"short": "CAR"
+					},
+					"constitution": {
+						"long": "constitución",
+						"short": "CON"
+					},
+					"default": {
+						"long": "Predeterminado",
+						"short": "Predeterminado"
+					},
+					"dexterity": {
+						"long": "Destreza",
+						"short": "DES"
+					},
+					"intelligence": {
+						"long": "Inteligencia",
+						"short": "INT"
+					},
+					"strength": {
+						"long": "Fuerza",
+						"short": "FUE"
+					},
+					"wisdom": {
+						"long": "Sabiduría",
+						"short": "SAB"
+					}
+				},
+				"baseUsageDie": "Dado de uso",
+				"brokenDice": "Dados de rotura",
+				"className": "Clase",
+				"coinsValue": "Monedas",
+				"currentUsageDie": "Dado actual",
+				"damage": "Daño",
+				"description": "Descripción",
+				"dice": "Dados",
+				"exhausted": "Agotado",
+				"hitDice": "Dados de golpe",
+				"hitDie": "Dado de golpe",
+				"hitPoints": "Puntos de golpe",
+				"hitPointsMax": "PG máximos",
+				"hitPointsValue": "PG actuales",
+				"key": "Key",
+				"largeWeapon": "¿Arma grande?",
+				"level": "Nivel",
+				"magicAttribute": "Característica",
+				"magicKind": "Tipo",
+				"magicLevel": "Nivel",
+				"name": "Nombre",
+				"none": "Ninguno",
+				"notApplicable": "-",
+				"notes": "Notas",
+				"playerName": "Nombre del jugador",
+				"quantity": "Cantidad",
+				"range": "Alcance",
+				"rarity": "Rareza",
+				"size": "Tamaño",
+				"tag": "Etiqueta",
+				"targetQuantity": "Target Quantity",
```

Comment:
This looks like it needs translating.

Reply:
Don't know if it helps but this comes from creature attacks and refers to the number of targets that can be hit by the attack (can be a fixed number or a roll).